pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "wheel", "numpy", "cython"]
```

Comment:
I don't think numpy and cython are build-system requirements -- they are build requirements, but that goes elsewhere.